app-web/wrapWithProvider.js

```diff
@@ -20,9 +20,21 @@ import React from 'react';
 import { ThemeProvider } from 'emotion-theming';
 import { AuthProvider } from './src/AuthContext';
 import theme from './theme';
+import { ApolloClient, InMemoryCache } from 'apollo-boost';
+import { ApolloProvider } from 'react-apollo';
+import { createHttpLink } from 'apollo-link-http';
+
+const cache = new InMemoryCache();
+//this needs to be an enviroment variable later on......
+const client = new ApolloClient({
+  link: createHttpLink({ uri: 'https://rocketgate-dev-devhub-dev.pathfinder.gov.bc.ca' }),
```

Comment:
this needs to come in as a config similar to how `auth.js` does it. It will either be an environment variable or a caddy template string. Checkout `config.js`